bot/exts/evergreen/status_codes.py

```diff
@@ -1,25 +1,34 @@
 from http import HTTPStatus
+from random import choice
 
 import discord
 from discord.ext import commands
 
-from bot.utils.extensions import invoke_help_command
-
 HTTP_DOG_URL = "https://httpstatusdogs.com/img/{code}.jpg"
 HTTP_CAT_URL = "https://http.cat/{code}.jpg"
 
 
 class HTTPStatusCodes(commands.Cog):
-    """Commands that give HTTP statuses described and visualized by cats and dogs."""
+    """
+    Commands that give HTTP statuses described and visualized by cats and dogs.
+
+    Give a cat or dog visualization randomly if the option is not filled.
+    """
```

Comment:
I think we can tidy up these docs a little.
```suggestion
    """
    Fetch an image depicting HTTP status codes as a dog or a cat.

    If neither animal is selected a cat or dog is chosen randomly for the given status code.
    """
```